fdbclient/admin_client.go

```diff
@@ -916,8 +916,7 @@ func (client *cliAdminClient) GetBackupStatus() (*fdbv1beta2.FoundationDBLiveBac
 // StartRestore starts a new restore.
 func (client *cliAdminClient) StartRestore(
 	url string,
-	keyRanges []fdbv1beta2.FoundationDBKeyRange,
-	encryptionKeyPath string,
+	fdbRestoreSpec fdbv1beta2.FoundationDBRestoreSpec,
```

Comment:
Is there a reason to pass down the spec and not the `*fdbv1beta2.FoundationDBRestore`? I would prefer to pass down the `*fdbv1beta2.FoundationDBRestore`.